package.json

```diff
@@ -37,6 +39,7 @@
     "spin": "nodemon --watch content --watch static --watch templates --ext md,rhai,hbs,css,js --verbose --legacy-watch --signal SIGINT --exec 'npm run build-index && spin up --file spin.toml'",
     "styles": "parcel watch static/sass/styles.scss --dist-dir static/css",
     "test": "npx markdownlint-cli2 content/**/*.md \"#node_modules\"",
-    "build-index": "node md_parser.mjs --dir=./content/ --out=./static/data.json"
+    "build-index": "node md_parser.mjs --dir=./content/ --out=./static/data.json",
+    "check-broken-links": ".build/check-broken-links.sh"
```

Comment:
Can we add this to the the `npm test` command? So that it runs automatically along with markdown linter. Thanks!

Reply:
done